hazenlib/utils.py

```diff
@@ -328,6 +329,69 @@ def get_image_orientation(iop):
         return "Transverse"
 
 
+def determine_orientation(dcm_list):
```

Comment:
This is not checking the phantom orientation because that depends on how the user put the phantom in the scanner. The `ImageOrientationPatient` tag tells you the orientation of the _imaging volume_ which contains the phantom. If the 'transverse' orientation of the phantom is aligned with the transverse definition of the imaging volume (e.g. [1, 0, 0, 0, 1, 0] in the comments below) then it couples that the phantom is in the transverse orientation.

However, the user could place the phantom at 90 degrees to 'transverse', but the same imaging volume might be applied. In this case, [1, 0, 0, 0, 1, 0] still reflects a transverse imaging volume, but this does not match the phantom. You have no way of knowing the **true** orientation of the phantom from the DICOM metadata.

There is also an added complication in all of this because different scanner vendors define there scanners axes in different ways. The FH (Foot-Head) direction on a Philips scanner might be defined as HF (Head-Foot) on a Siemens scanner.

My feeling is that if the unit tests are passing with the code below, then keep it in, assuming it doesn't affect anything else in hazen. But be mindful that this could be a source of bugs in the future. Which, might have also been present with the previous implementation.

Reply:
some of this caveat did become an apparent problem and is fixed in #416